src/components/SearchModal/PlacesAutocomplete.tsx

```diff
@@ -25,14 +33,37 @@ const PlacesAutocomplete = () => {
     clearSuggestions();
   };
 
+  const handleSubmit = () => {
+    setIsOpen(false);
+    router.push({
+      pathname: '/post/travelogueList',
+      query: { keyword: value },
+    });
+  };
+
+  const handleKeyDown = (e: KeyboardEvent<HTMLDivElement>) => {
+    if (e.key === 'Enter') {
+      e.preventDefault();
+      handleSubmit();
+    }
+  }
```

Comment:
p1;

한글일 경우 검색 버튼을 클릭할 때는 괜찮은데 엔터를 누르면 끝 글자가 한 글자 더 추가되고 있는데 수정하셔야 할 것 같습니다.
![image](https://user-images.githubusercontent.com/63948884/223754876-b6128e42-22be-4b90-b95e-cb095e1acefe.png)

Reply:
```suggestion
  const handleKeyDown = (e: KeyboardEvent<HTMLDivElement>) => {
  if (e.nativeEvent.isComposing) return;
    if (e.key === 'Enter') {
      e.preventDefault();
      handleSubmit();
    }
  }
```
[[참고글](https://velog.io/@corinthionia/JS-keydown%EC%97%90%EC%84%9C-%ED%95%9C%EA%B8%80-%EC%9E%85%EB%A0%A5-%EC%8B%9C-%EB%A7%88%EC%A7%80%EB%A7%89-%EC%9D%8C%EC%A0%88%EC%9D%B4-%EC%A4%91%EB%B3%B5-%EC%9E%85%EB%A0%A5%EB%90%98%EB%8A%94-%EA%B2%BD%EC%9A%B0-%ED%95%A8%EC%88%98%EA%B0%80-%EB%91%90-%EB%B2%88-%EC%8B%A4%ED%96%89%EB%90%98%EB%8A%94-%EA%B2%BD%EC%9A%B0)]
한글은 조합 문자여서 입력 중인 마지막 글자 바로 아래에 밑줄이 생기는데, 이 경우 엔터를 입력하면 이벤트가 두 번 발생한다고 합니다. 
`KeyboardEvent.isComposing`은 입력한 문자가 조합문자인지 아닌지를 판단한다고 합니다.
위 함수를 사용하면 마지막 글자가 한 번 더 입력되는 상황을 방지할 수 있을 것 같습니다.

Reply:
맥북환경에서 고려를 못했네요...
윈도우에서는 문제가 없어서 처음본 이슈입니다.
감사합니다!